plugin/inc/task_backup.class.php

```diff
@@ -98,44 +97,49 @@ protected function restoreDB($backup_file_path)
         return $this->getResponseDiv("Restore sucessfull" . ($import_failed ?? '') . ".");
     }
 
-    protected function displayBackups($backups = [])
+    protected function getBackupsList($backups = [])
```

Comment:
This function does not return a list, but a html!
```suggestion
    protected function getBackupsListHtml($backups = [])
```